### DISCUSSION TITLE
Form decision & Infinite scrolling example

### DISCUSSION THREAD

**Discussion by Gallevy** [epicweb-dev/epic-stack#723] [Q&A] [answered]
Hey, I just came across this stack and I'm still trying to put the puzzle pieces together. So far it looks amazing but a few things I missed, maybe its there but I didn't notice:

- Form decision (Why conform and not something else)
- Infinite scroll/loading example as this is leans heavily into client state which seems to not be the remix philosophy

**Comment by kentcdodds** [comment, marked as answer]
Hi @Gallevy,

I haven't had the time to write up a decision doc on conform, but it was a very thoughtful choice. I've tried everything else and this one is the best.

Infinite scroll is definitely a thing you can do with remix and it's not really against the remix philosophy (the remix philosophy is "excellent UX"). You could probably find some remix examples online and apply those things in the Epic Stack without issue.

Good luck!

**Reply by lewisblackburn** [reply]
I have an InfiniteScroll component that might be useful here: https://github.com/lewisblackburn/petal/blob/main/app/components/infinite-scroll.tsx.

This can be used in addition to the history state.

**Reply by Gallevy** [reply]
Thanks both! super helpful.
